### DISCUSSION TITLE
0.0.1.63: Новые адреса для АнтиЗапрета. Последний выпуск в формате Manifest V2

### DISCUSSION THREAD

**Discussion by ilyaigpetrov** [anticensority/runet-censorship-bypass#226] [General]
Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

**Comment by ShohninDmitriy** [comment]
Установил в FireFox полет нормальный

**Comment by rokozh** [comment]
В chrome  (123.0.6312.86) - работает!

**Comment by Verity-Freedom** [comment]
Мы также подготовили портативный Тор для режима антицензорити на случай если (не дай Бог) вновь произойдёт нечто подобное. В режиме антизапрета он позволит свободно открывать сайты onion на любом браузере или подключаться к чему-нибудь через Тор по исключениям (при желании).

Посмотрите, работает ли, можете написать свой отклик.
https://github.com/anticensority/runet-censorship-bypass/issues/229

**Reply by megafanat** [reply]
Замечание к инструкции к настройке расширения для работы через Tor:

> 1) Убедитесь, что в расширении стоят флаги: учитывать исключения (в разделе "Исключения"), Использовать прокси PAC-скрипта, Использовать СВОЙ локальный Tor, СВОИ прокси только для СВОИХ сайтов, Проксировать .onion (В разделе "Свои прокси"). Используйте или режим антизапрет, или режим антицензорити (совместимо с обоими). В режиме "антизапрет" через Тор будут проксироваться только сайты-исключения (проксировать - да), в режиме "антицензорити" проксироваться через него будут все сайты.

Ничего не понятно. Где это? Что это? Можно скриншот с экраном настроек приложить?

**Reply by rokozh** [reply]
На win10 заработало

**Comment by Verity-Freedom** [comment]
По моим впечатлениям версия 1.63 критических для функционала багов не имеет, о том, что заметил подозрительного - отрапортовал. Можете скачивать спокойно, хуже не будет.

**Comment by akimtsev** [comment]
в яндекс браузер работает хорошо

**Comment by ShohninDmitriy** [comment]
Осталось только подписать, а то в FireFox приходиться каждый раз загружать расширение

**Reply by ilyaigpetrov** [reply]
Для подписи нужно пройти проверку работниками Mozilla, review которая. Та же проверка, что и при публикации расширения, только вместо страницы в магазине получаешь подписанный архив, ну или что-то подобное для  распространения через передачу файлов.

Спасибо вам и всем остальным за участие в тесте.

**Comment by BavrodyaDEV** [comment]
Поставил на Chrome, ютуб и твиттер пашут

**Comment by gremilkar** [comment]
Вроде работает,Google Chrome Windows 11 x64.

**Comment by hada1987** [comment]
Яндекс браузер, отлично работает!

**Comment by Trergfirestorm** [comment]
В хроме работает отлично пока что. А для OpenVPN на  Android  есть варианты решения?

**Reply by ghost** [reply]
См. https://ntc.party/t/новый-адрес-pac-файла-и-хоста-vpn/7610.

**Comment by Verity-Freedom** [comment]
Будут ли фиксы у релизной версии 1.63 по сравнению с пререлизной или никаких изменений не планируется? Чтобы знать стоит ли переустановить пререлиз 1.63 на релиз.

**Reply by ghost** [reply]
Изменения будут, если ревьюеры магазинов потребуют их внести.
Я бы рекомендовал использовать автономную версию (из архива) только как временный вариант до публикации в магазине, т.к. в автономной версии не будет обновляться автоматически сам код расширения, что может привести к бажному поведению.

**Reply by Verity-Freedom** [reply]
Ну вообще хочу отметить что я и до этого пользовался автономной версией 1.62 с Гитхаба как раз из-за того что на Гитхабе оно обновилось первее чем в магазине...

Но хороший аргумент.

Я больше имел ввиду тот факт что выявились ли баги первого дня из-за которых автономную версию лучше переустановить или не выявились.

А хромые могут потребовать переписывать на манифест третий, тогда отпадёт совместимость со всеми хромиумами вплоть до 86-го, о таком желательно будет предупредить заранее.

**Reply by ghost** [reply]
> А хромые могут потребовать переписывать на манифест третий[...]

Я постараюсь опубликовать обновление в формате Manifest V2. Планируется, что это будет последний релиз на данном API, за которым последуют исправления мелких багов и затем MV3.

**Reply by VisualKei** [reply]
Очень прошу не забрасывать совсем версию MV2, а по возможности обновлять параллельно с MV3, если что будет критического. Думаю не только у меня есть старые, но всё ещё используемые машины с WinXP, где последний по "свежести" доступный браузер это 360EE 13.5 на Chrome 86, соответственно без поддержки MV3. Возможно и на более новых ОС кто-то предпочитает использовать старые версии Chrome. Было бы очень печально лишиться там такого простого и замечательного антицензурного средства - сопротивление цензуре должно быть возможно даже в арьергарде. И я подчёркиваю, что не прошу поддерживать две полностью одинаковые по функционалу версии, а просто продолжить фиксить и выкладывать MV2-версию, если она перестаёт работать по известным причинам.

**Comment by AlesFerum** [comment]
Спасибо за труды!

**Comment by Perkas-Vlad** [comment]
Спасибо, всё работает

**Comment by Narkoman4ek** [comment]
В OperaGx отлично работает, только в списке расширений ползунок включения/выключения вылезает за рамки окна расширения, а так всё прекрасно
![image](https://github.com/anticensority/runet-censorship-bypass/assets/40146638/d6276463-0018-4ed0-a6d3-ef3cee8772e6)

**Reply by Verity-Freedom** [reply]
Проверь было ли так на предыдущей версии

**Reply by Narkoman4ek** [reply]
Не было
![image](https://github.com/anticensority/runet-censorship-bypass/assets/40146638/75ab18d2-0eb4-4b2f-b4ec-ff91c8d80d90)

**Reply by Verity-Freedom** [reply]
Ответил ниже.

**Comment by Verity-Freedom** [comment]
@Narkoman4ek у тебя написано что есть какие-то ошибки, багрепорт делай.

**Reply by Narkoman4ek** [reply]
Подскажи как, я не очень разбираюсь в этом))

**Reply by Verity-Freedom** [reply]
Кликни на надпись "ошибки" и скопируй что пишут, а дальше сделай баг-репорт в issues.

**Reply by Verity-Freedom** [reply]
Про съехавший интерфейс тоже напиши, раз так не было на предыдущей версии, то так не должно быть.

**Reply by Verity-Freedom** [reply]
Issues это вот что
https://github.com/anticensority/runet-censorship-bypass/issues

**Reply by Narkoman4ek** [reply]
Надеюсь более менее правильно оформил

**Reply by Verity-Freedom** [reply]
Тут не особо важно ибо у меня тоже такое есть.

**Comment by Verity-Freedom** [comment]
Мы же бетатестеры по сути

**Comment by Chalm33rs** [comment]
Ребята это база!! спасибо вам огромнейшее за то, что вы есть!!! Можете, пожалуйста скомпилировать OpenVPN конфигурацию?

**Reply by Verity-Freedom** [reply]
У нас скомпилирована только конфигурация на Торе. На open vpn конфигурации для расширения нет, можете сами заняться, если разбираетесь. Но разработчик расширения советует перейти по этой ссылке, создатели самого прокси и расширения это люди немного разные.
https://ntc.party/t/%D0%BD%D0%BE%D0%B2%D1%8B%D0%B9-%D0%B0%D0%B4%D1%80%D0%B5%D1%81-pac-%D1%84%D0%B0%D0%B9%D0%BB%D0%B0-%D0%B8-%D1%85%D0%BE%D1%81%D1%82%D0%B0-vpn/7610/3

**Reply by Chalm33rs** [reply]
Скачал новую конфигурацию по данной ссылке и всё заработало!!

**Comment by AlenaJaris** [comment]
Chrome версия 123.0.6312.86 (Официальная сборка), (64 бит) работает отлично.

**Comment by DanteDragan** [comment]
Vivaldi, Arch Linux, Ру****ер грузится мгновенно. Спасибо!!!

**Comment by rellikmil** [comment]
Последний официальный Firefox, заработало через расширение

**Comment by Pr1nkos** [comment]
Opera beta 109 - все работает

**Comment by PQV0jCT** [comment]
Браузер Comodo Dragon. Version 121.0.6167.140 (64 бит)
Работает как раньше но в управление  расширений показывает две ошибки. 
Unrecognized manifest key 'browser_specific_settings'.
{
  "manifest_version": 2,

  "name": "__MSG_extName__",
  "default_locale": "ru",
  "description": "__MSG_extDesc__",
  "version": "0.0.1.63",
  "icons": {
    "128": "icons/default-128.png"
  },
  "author": "anticensority+owners@googlegroups.com",

  "permissions": [
      "proxy"
    , "alarms"
    , "storage"
    , "unlimitedStorage"
    , "<all_urls>"
    , "tabs"
    , "contextMenus"
    , "notifications"
    , "webRequest", "webRequestBlocking", "webNavigation"
  ],
  "minimum_chrome_version": "55.0.0.0",
  "browser_specific_settings": {
    "gecko": {
      "strict_min_version": "91.1.0"
    }
  },

  "background": {
    
    "scripts": [
        "00-init-apis.js"
      
      , "11-error-handlers-api.js"
      , "12-errors-lib.js"
      , "13-http-lib.js"
      , "15-firefox-proxy-settings.js"
      , "20-ip-to-host-api.js"
      , "35-pac-kitchen-api.js"
      , "37-sync-pac-script-with-pac-provider-api.js"
      , "80-error-menu.js", "83-last-errors.js", "85-block-informer.js"
      , "70-menu-items.js"
      , "75-context-menus.js"
    ]
  },

  "browser_action": {
    "default_title": "Этот сайт благословлён | Версия 1.63",
    "default_popup": "/pages/options/index.html"
  },
  "options_ui": {
     "page": "/pages/options/index.html",
     "chrome_style": false
   }

}

И вторая Manifest version 2 is deprecated, and support will be removed in 2024. See https://developer.chrome.com/docs/extensions/migrating/mv2-sunset/ for more details.

{
  "manifest_version": 2,

  "name": "__MSG_extName__",
  "default_locale": "ru",
  "description": "__MSG_extDesc__",
  "version": "0.0.1.63",
  "icons": {
    "128": "icons/default-128.png"
  },
  "author": "anticensority+owners@googlegroups.com",

  "permissions": [
      "proxy"
    , "alarms"
    , "storage"
    , "unlimitedStorage"
    , "<all_urls>"
    , "tabs"
    , "contextMenus"
    , "notifications"
    , "webRequest", "webRequestBlocking", "webNavigation"
  ],
  "minimum_chrome_version": "55.0.0.0",
  "browser_specific_settings": {
    "gecko": {
      "strict_min_version": "91.1.0"
    }
  },

  "background": {
    
    "scripts": [
        "00-init-apis.js"
      
      , "11-error-handlers-api.js"
      , "12-errors-lib.js"
      , "13-http-lib.js"
      , "15-firefox-proxy-settings.js"
      , "20-ip-to-host-api.js"
      , "35-pac-kitchen-api.js"
      , "37-sync-pac-script-with-pac-provider-api.js"
      , "80-error-menu.js", "83-last-errors.js", "85-block-informer.js"
      , "70-menu-items.js"
      , "75-context-menus.js"
    ]
  },

  "browser_action": {
    "default_title": "Этот сайт благословлён | Версия 1.63",
    "default_popup": "/pages/options/index.html"
  },
  "options_ui": {
     "page": "/pages/options/index.html",
     "chrome_style": false
   }

}

**Reply by ilyaigpetrov** [reply]
Это предупреждения, не ошибки. Их можно игнорировать.

**Comment by bauglir616** [comment]
Спасибо, супер, все работает!

**Comment by 1600w** [comment]
спасибо за расширение

**Comment by Skaamit** [comment]
Ростелеком - Chrome  123.0.6312.86 - Fedora Linux 39

Всё работает хорошо. 👌

Огромное спасибо и желаю вам успехов в разработке.

**Comment by NTMan** [comment]
3 предупреждения было найдено.

Unrecognized manifest key 'browser_specific_settings'.
![Screenshot from 2024-03-30 17-13-13](https://github.com/anticensority/runet-censorship-bypass/assets/200750/6f232b14-1512-48e9-8912-43823963f35e)

Manifest version 2 is deprecated, and support will be removed in 2024. See https://developer.chrome.com/docs/extensions/develop/migrate/mv2-deprecation-timeline for details.
![Screenshot from 2024-03-30 17-13-07](https://github.com/anticensority/runet-censorship-bypass/assets/200750/5f11f38d-57e5-4748-82ab-abd231273a10)

Third-party cookie will be blocked. Learn more in the Issues tab.
![Screenshot from 2024-03-30 17-13-00](https://github.com/anticensority/runet-censorship-bypass/assets/200750/a6f29ca5-0c5a-47bd-b0ef-c85d26d4b936)

Броузер Google Chrome Dev version 125.0.6382.3 (Official Build) dev (64-bit)

**Reply by Verity-Freedom** [reply]
Уже багрепорты отправлены.

Не всё из этого достойно внимания. Возможно вообще ничего из этого не достойно внимания.

**Comment by ZloyNord** [comment]
Заменил в Firefox - работает. Спасибо, разработчик(и)!

**Comment by Verity-Freedom** [comment]
Для тех, кто беспокоился. Выяснилось, как убрать "ошибки" расширения.
https://github.com/anticensority/runet-censorship-bypass/issues/233

**Comment by Yggaz** [comment]
Сегодня, 31 марта, 1.63 перестало работать в Google Chrome.
При этом 1.61 сегодня же начало работать снова (переставало вчера).
Думаю, это что-то с провайдером, но не очень знаю, стоит ли что-то рассказывать подробнее и если да, то что именно.

**Reply by Verity-Freedom** [reply]
Гугл Хром постоянно обновляется в тихом режиме. Они могут втихую отключить доступ к своему браузеру россиянам и не поперхнуться. Разумеется может сломаться всё что угодно.

1.61 стабильно работать больше не будет, так как адреса, на которые он опирается, находятся в реестре.

**Comment by madararara** [comment]
Установил версию 1.63, все работает, но dropstab.com почему-то перестал полноценно загружаться. При включении какого-либо vpn все грузится нормально. Я так понимаю эта проблема не связана с недавней блокировкой антизапрета, т.к. ранее сайт грузился нормально? Если все же связано и что-то блокируют с нашей стороны, можно ли добавить доступ к сайту?

**Reply by ilyaigpetrov** [reply]
Этого адреса нет в реестре: https://reestr.rublacklist.net/ru/?q=dropstab.
Также мой провайдер его не блокирует.
Пришлите в группу поддержки https://groups.google.com/g/anticensority скриншот этой неисправности.

**Reply by ShohninDmitriy** [reply]
Так в исключения добавь просто

**Reply by Verity-Freedom** [reply]
У меня на антизапрете всё с ним нормально.

**Comment by ilyaigpetrov** [comment]
1.63 опубликована в WebStore.
Новость: https://github.com/anticensority/runet-censorship-bypass/issues/10#issuecomment-2029542840
Версия в WebStore: https://chromewebstore.google.com/detail/npgcnondjocldhldegnakemclmfkngch

**Comment by localghost1817** [comment]
![image](https://github.com/anticensority/runet-censorship-bypass/assets/98688483/eaacbbd7-9c18-4fee-805a-65be60f5039c)
пофиксите плз thebyrut

**Reply by ilyaigpetrov** [reply]
Сайт блокирует наши прокси-сервера, не знаю: умышленно или нет. Вы можете передать админам эту ссылку с адресами наших прокси для разблокировки: https://antizapret.prostovpn.org:8443/do-not-block-this-proxy/.

**Reply by localghost1817** [reply]
Знать бы еще как с ними связаться, на сайт их не попасть, контактов нет в поисковиках, ни вк, ни тг. Хз в общем))

**Reply by Verity-Freedom** [reply]
Byrut это явно не топ один сайт который я бы советовал. Я ловил на нём очень криповый вирус. См. https://www.virustotal.com/gui/file/5f1c46da9e266b1f7f31953e593636c6b0e1968636e81e62e820122bfa40706f

**Reply by localghost1817** [reply]
Themida - коммерческая утилита-протектор. Ее цель - защитить программный код от его изучения, модификации, взлома путем запутывания. Если исходный код, написанный программистом, и полученный после компилляции, относительно легко читаем в программе-дизассемблере (соответственно, его легко изменить, "взломать"), то тот же код, обработанный Themida, превращается в "кашу", разобраться в которой довольно трудно.

Применяя Themida автор полезной программы может существенно усложнить ее взлом, защитив тем самым свою программу от пиратства.

Алгоритм "распутывания" сложен, да и сама дешифровка занимает относительно много времени.
Некоторые антивирусы поддерживают Themida и могут добраться до распутанного содержимого, а некоторые поступают проще: по некоторым признакам определяют, что файл обработан Themida, и автоматически называют его вредным, не осложняя себе жизнь анализом собственно содержания файла.

🤔🤔🤔

**Reply by Verity-Freedom** [reply]
Ну просто чтобы не быть голословным:
![image](https://github.com/anticensority/runet-censorship-bypass/assets/143919205/1521e7f9-f604-4c4b-b279-72d5d677818b)

На крайний случай его можно через Tor открыть без проблем.

**Reply by Verity-Freedom** [reply]
Так что скорее правильно, что РКН блокнул рассадник вирусов, а сами они ещё и от популярного прокси огородились. Но если прям нужно-нужно, то берите и ставьте автономный Тор по моему гайду, с ним всё работает, проверено.

**Reply by Verity-Freedom** [reply]
> по некоторым признакам определяют, что файл обработан Themida, и автоматически называют его вредным

Надеюсь вы не думаете, что файл, на Вирустотале определённый как вирус 60 из 70 раз, это false positive?

**Comment by SerIcy** [comment]
Какой нахуй Assets, блядь?! Где это?

**Reply by ilyaigpetrov** [reply]
На этой странице: https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63, внизу.

**Comment by Olezha180** [comment]
> Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

Ну что, когда снова заработает?

**Comment by Olezha180** [comment]
> Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

Facebook и Instagram не открываются

**Comment by silvercis** [comment]
Инстаграм не работает

**Comment by Olezha180** [comment]
Снова ничего не работает

**Comment by Olezha180** [comment]
> Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

Опять ничего не работает

**Comment by Olezha180** [comment]
> Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

Утром торрент-сайты и вебкам-сайты открывались, сейчас - нет, чё за фигня? Что происходит?

**Reply by rellikmil** [reply]
Вы, может, будете эти вопросы правительству задавать, которые на ваши же налоги вам запрещает сидеть где вам хочется? Мне уже надоело читать на почте ваше нытье, не работает - значит ждите, пока тут оперативно перебьют адреса, либо как я включайте Censor Tracker (он в чем-то похуже работает, но тем не менее на моей памяти он работает без прерываний)

**Reply by Olezha180** [reply]
А я откуда знаю, забросили вы или нет?

**Reply by rellikmil** [reply]
Я не разработчик вот этого всего, если вы об этом. Мне кажется, если забросят - то скажут.

**Reply by Verity-Freedom** [reply]
> Утром торрент-сайты и вебкам-сайты открывались, сейчас - нет, чё за фигня

Очень многие уже моим дополнением к расширению пользуются, которое ещё и Ютуб умеет проксировать.

@rellikmil у Censor Tracker рейтинг на Вебсторе ещё меньше, полно жалоб на неработоспособность. Без понятия, чем он лучше или хуже, но думаю, что на него в целом распространяется та же логика, что и сюда.

Upd. у Censor Tracker нет важного функционала совместного использования со своими прокси, или то, или другое. Но в целом сейчас и он, и Антизапрет/Обход работают.

**Reply by Verity-Freedom** [reply]
> А я откуда знаю, забросили вы или нет?

Читайте новости.

**Reply by Olezha180** [reply]
> > Утром торрент-сайты и вебкам-сайты открывались, сейчас - нет, чё за фигня
> 
> Очень многие уже моим дополнением к расширению пользуются, которое ещё и Ютуб умеет проксировать.
> 
> @rellikmil у Censor Tracker рейтинг на Вебсторе ещё меньше, полно жалоб на неработоспособность. Без понятия, чем он лучше или хуже, но думаю, что на него в целом распространяется та же логика, что и сюда.
> 
> Upd. у Censor Tracker нет важного функционала совместного использования со своими прокси, или то, или другое. Но в целом сейчас и он, и Антизапрет/Обход работают.

О чём речь, можно ссылку?

**Reply by Verity-Freedom** [reply]
> О чём речь, можно ссылку?

https://github.com/Verity-Freedom/Tor-Portable/

**Reply by rellikmil** [reply]
@Verity-Freedom я говорю про свой собственный опыт, лично у меня не было такого, что он не работает. Да, он работает многократно кривее и прочее, но работает. Когда AntiZapret все же начинает работать, я пользуюсь им, он быстрее и удобнее, вот что я хотел сказать.

**Reply by Verity-Freedom** [reply]
> я говорю про свой собственный опыт, лично у меня не было такого, что он не работает

Да я бы сказал что это неплохое расширение. В списке альтернатив официальном даже присутствует.

Сейчас автор Антизапрета работает над противостоянием блокировкам по ip, надеюсь скоро Обход станет работать стабильнее.

**Comment by Olezha180** [comment]
> Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

Ну и что? Сегодня 1-е октября, даже торрент-сайты не открываются

**Reply by Verity-Freedom** [reply]
Не будет ничего открываться пока сервера лежат. Видимо возникли сложности и потребуется или чуть больше времени, или вообще, всё плохо. Я подтверждаю тот факт, что всё лежит.

**Comment by SerIcy** [comment]
Что я тут вообще делаю?


вт, 1 окт. 2024 г. в 12:46, Verity-Freedom ***@***.***>:

**Reply by Olezha180** [reply]
В смысле?

**Comment by Olezha180** [comment]
> Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

Торрент-сайты не работают

**Reply by Verity-Freedom** [reply]
Не может что-то отдельно не работать. Или работает всё, или не работает ничего (кроме Тора, Тор всегда работает).

**Reply by Olezha180** [reply]
Если говорить про последнее время, то иногда торрент-сайты и вебкам-сайты нормально работают, а вот Instagram, Facebook и X - ВООБЩЕ НЕТ !

**Reply by Verity-Freedom** [reply]
Там обещали включить их 1 октября но видимо возникли сложности производственные.

**Comment by Olezha180** [comment]
> Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

Опять ничего не работает

**Comment by Olezha180** [comment]
> Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

У меня на телефоне стоит "Super VPN" (зелёная иконка у него), который вообще никак не банится РКН'ом. Почему ваше приложение тогда банится?

**Reply by ilyaigpetrov** [reply]
1) Наше расширение не является VPN-ом. Любое браузерное расширение, якобы являющееся VPN-ом, на самом деле работает на обычных прокси, часто вкупе с PAC-скриптом. Наше -- в том числе. Это ограничение на уровне программного интерфейса (API) для расширений в Хромиумах и FireFox, так что их работа с VPN возможна только вне браузера, т.е. требуется установка дополнительных программ помимо браузера.

2) РКН пока не банит во всю свою силу настоящие VPN-ы в РФ. Я пока не знаю точной причины. Возможно, они дают гражданам время, чтобы привыкнуть (подход гомеопатии) и смириться, не вызывая резкой негативной реакции и протестов. Возможно, механизмы блокировки VPN предполагают запрет всего трафика за границу (режим белого списка для иностранных адресов), к которому они пока не готовы сами (технически) и не подготовили граждан (морально-психологически).

**Reply by Verity-Freedom** [reply]
У меня в последние дни работает более-менее стабильно.

**Reply by Olezha180** [reply]
> У меня в последние дни работает более-менее стабильно.

А у меня нет

**Comment by ShohninDmitriy** [comment]
Привет всем, с discord что-то сделать а то через аддон не открывает после блокировки

**Reply by Verity-Freedom** [reply]
В портативном Торе можно прописать его адреса вручную или переключаться на режим проксирования им всего трафика если использовать SwitchyOmega, но это не имеет большого смысла, ибо войс это не разблочит. К тому же Дискорд - приложение, а это - браузерное расширение.

**Comment by Olezha180** [comment]
> Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

Опяяять Инста не работает

**Comment by Olezha180** [comment]
> Обсуждение [1.63](https://github.com/anticensority/runet-censorship-bypass/releases/tag/0.0.1.63).

Снова Инстаграм не работает

**Reply by ilyaigpetrov** [reply]
Пришлите снимок проблемы, с которой вы сталкиваетесь (ошибка, поломанный интерфейс и т.п.).

**Reply by Olezha180** [reply]
> Пришлите снимок проблемы, с которой вы сталкиваетесь (ошибка, поломанный интерфейс и т.п.).

А, всё, заработало, очень странно

**Reply by ilyaigpetrov** [reply]
В следующий раз создайте новое обсуждение. Вот так:
![rcb-new-discussion](https://github.com/user-attachments/assets/271194b9-3434-46b8-9a70-95539b785f48)

**Reply by Olezha180** [reply]
> В следующий раз создайте новое обсуждение. Вот так: ![rcb-new-discussion](https://private-user-images.githubusercontent.com/1090811/395854990-271194b9-3434-46b8-9a70-95539b785f48.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUwNTE3NTAsIm5iZiI6MTczNTA1MTQ1MCwicGF0aCI6Ii8xMDkwODExLzM5NTg1NDk5MC0yNzExOTRiOS0zNDM0LTQ2YjgtOWE3MC05NTUzOWI3ODVmNDgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMjRUMTQ0NDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2Q3NGQyYzdhOTYyZTgyMTM3OGQ3MGYzNzc0ZDNlNDI0OWFjYjQxMGZlYTc0NTFlYTAzYzAwNmIwZWU4N2NlYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.BJfd4J_Up3HxvjirH7IcKQ6KYc1PQ4xjdARffrW8l2g)

Facebook снова не работает

**Reply by Olezha180** [reply]
> В следующий раз создайте новое обсуждение. Вот так: ![rcb-new-discussion](https://private-user-images.githubusercontent.com/1090811/395854990-271194b9-3434-46b8-9a70-95539b785f48.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDk1NzIzMDYsIm5iZiI6MTc0OTU3MjAwNiwicGF0aCI6Ii8xMDkwODExLzM5NTg1NDk5MC0yNzExOTRiOS0zNDM0LTQ2YjgtOWE3MC05NTUzOWI3ODVmNDgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDYxMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA2MTBUMTYxMzI2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGUzY2ZhMTRjMjk4MGVjNTBiM2JhYzZjYTczOTQ5MTE5MTMxODRmYjkzYzJlOTk3OTA3YTIzNDU0YjA3YjVjNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.1-j_-5KUWHSUf1imj9oUg7-EDRoz-999LmW8Q2Zhdsg)

Здравствуйте! Расширение снова не работает

**Comment by jjanelane** [comment]
В Хроме перестало работать, невозможно зайти на Linkedin.

**Comment by KSGiNET** [comment]
Manifest version 2 is deprecated, and support will be removed in 2024. See https://developer.chrome.com/docs/extensions/develop/migrate/mv2-deprecation-timeline for details.
{
  "manifest_version": 2,

**Comment by wonordel** [comment]
ошибка
![image](https://github.com/user-attachments/assets/33df3e1b-fc6d-49ce-acbc-967da7ea3d1d)

**Reply by sx007** [reply]
https://github.com/anticensority/runet-censorship-bypass/discussions/386
Глянь

**Comment by rom23rom** [comment]
Сегодня обновился Хром до версии 140 и расширение полностью отвалилось.

**Reply by exwound** [reply]
Такое ощущение что проект мёртв, автор ничего не обновляет, не отвечает.

**Comment by DemonTwR** [comment]
Родные да сделайте вы уже какой то Форк что бы на Chrome & Chronium версия 0.0.1.64 работала без режима разраба ну это просто не реально неудобно .
И кстати я в шоке почему обнова 0.0.1.67 и 68 только на edge который как бы тоже на chrome основан что за Дискриминация ! ! !
А касательно комента [exwound](https://github.com/exwound)
[on Sep 30](https://github.com/anticensority/runet-censorship-bypass/discussions/226#discussioncomment-14548956) ну так может до автора добрались Люди с "Новичком" ...

**Reply by Verity-Freedom** [reply]
> Родные да сделайте вы уже какой то Форк что бы на Chrome & Chronium версия 0.0.1.64 работала без режима разраба ну это просто не реально неудобно . И кстати я в шоке почему обнова 0.0.1.67 и 68 только на edge который как бы тоже на chrome основан что за Дискриминация ! ! ! А касательно комента [exwound](https://github.com/exwound) [on Sep 30](https://github.com/anticensority/runet-censorship-bypass/discussions/226#discussioncomment-14548956) ну так может до автора добрались Люди с "Новичком" ...

Идейный наследник расширения: https://github.com/Verity-Freedom/Tor-Portable

**Comment by musya837-tech** [comment]
Всем привет!
Пишу из апреля 2026г. - на win11 в хроме невозможно установить расширение. При установке его пишет
<img width="948" height="76" alt="Снимок экрана" src="https://github.com/user-attachments/assets/de03bd6c-523f-4c6c-8724-0075c31b5e7c" />
Помогите, пжл. Расширение для фаерфокс работает. Устанавливаю файл с названием - runet-censorship-bypass-0.0.1.63-full-rc0.crx

**Comment by passmiss1** [comment]
15.06.26 Что произошло, у меня запрет сам включился в хроме и работает, даже на рутрекер и твиттер заходит :)
